client/question/services/FeedbackGeneratorService.js

```diff
@@ -13,9 +13,9 @@
 // limitations under the License.
 
 /**
- * @fileoverview Service for generating feedback based on the evaluation result
- * of the code submitted by the user, preprocessed and augmented with unit
- * tests. This assumes that the base code already passes "compile-time" checks.
+ * @fileoverview Service for creating human-readable feedback messages based
```

Comment:
I think "more informative (friendly) error messages" would be better. "human-readable" implies the original errors were not human-readable, when they are (they just aren't very informative or friendly to novices).

Reply:
Hm, I think that's not the right context for this fileoverview, though, which isn't restricted to just syntax errors. In general, what the functions in this file do is that they transform a FeedbackDetails object (which is, in general, not human-readable) to something that's human readable.

Reply:
I see. I misunderstood our offline chat. OK.